apps/vc-api/docs/example-exchange-definitions/presentation-issuance-self-sign-exchange.md

```diff
@@ -0,0 +1,95 @@
+Sample Exchange Definition for a issuance, self-sign and presentation.
+
+Use case:
+
+1. The Holder is required to present a self-signed consent VC and a PermanentResidentCard VC (Issued by Authority) to obtain another Verifiable Credential from the Issuer.
```

Comment:
As each example only has one use case, I don't think the enumeration is necessary. What do you think?
```suggestion
Use case: The Holder is required to present a self-signed consent VC and a PermanentResidentCard VC (Issued by Authority) to obtain another Verifiable Credential from the Issuer.
```